Sources/MessagingInApp/MessagingInAppImplementation.swift

```diff
@@ -4,26 +4,38 @@ import Foundation
 import Gist
 
 internal class MessagingInAppImplementation: MessagingInAppInstance {
+    private let siteId: SiteId
+    private let region: Region
```

Comment:
I believe we also talked about, adding `debug` state which corresponds to our loglevel